UM-USElab/gradecraft-development
# Add recurring course export job

### Status
**READY**

### Description
Adds a monthly recurring job that exports all courses created within the last month to CSV and sends it to the Gradecraft admins.

As with all Resque jobs, an admin should be able to access the dashboard to manually trigger this job at any time by going to `/jobs`

### Gem dependencies
N/A

### Migrations
NO

### Steps to Test or Reproduce
Ensure that the export produces results as expected for the month prior.

======================
Closes #3988
